taqueria-vscode-extension-web-ui/src/data-editors/MapEditor.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { DataEditorNode } from './DataEditorNode';
+import { getFriendlyDataType } from './MichelineEditor';
+import { compare, validate } from './MichelineValidator';
+import { ValidationResultDisplay } from './ValidationResultDisplay';
+import { VSCodeButton } from './VsCodeWebViewUIToolkitWrappers';
+
+export const MapEditor = (
+	{ dataType, value, onChange }: { dataType: any; value: any; onChange: (value: any) => void },
+) => {
+	if (value === undefined || value === null || !Array.isArray(value)) {
+		value = [];
+		onChange(value);
+	}
+	const changeValue = (index: number, key: any, v: any) => {
+		const newValue = value.slice();
+		newValue[index] = {
+			'prim': 'Elt',
+			args: [
+				key,
+				v,
+			],
+		};
+		onChange(newValue);
+	};
+	const remove = (index: number) => {
+		const newValue = value.slice();
+		newValue.splice(index, 1);
+		onChange(newValue);
+	};
+	const sort = () => {
```

Comment:
Is this meant to sort in reverse order? If so, naming it `sortInDescending` makes more sense.

e.g. `const sortInDescending = () => ...`

Reply:
Please see the comment above.